core/org.wso2.carbon.ui/src/main/resources/web/admin/login.jsp

```diff
@@ -196,6 +216,15 @@ if (CharacterEncoder.getSafeText(request.getParameter("skipLoginPage"))!=null){
 
                         <form action='../admin/login_action.jsp' method="POST" onsubmit="return doValidation();" target="_self" onsubmit="checkInputs()">
                             <table>
+                                <%if (enableBanner) { %>
```

Comment:
This is not null safe because a null type cannot be converted to a boolean. So, let's define a boolean value to this at L64